engine-tests/src/test/java/org/terasology/engine/ComponentSystemManagerTest.java

```diff
@@ -0,0 +1,70 @@
+
+package org.terasology.engine;
+
+import com.google.common.collect.Iterables;
+import org.junit.Before;
+import org.junit.Test;
+import org.terasology.context.Context;
+import org.terasology.entitySystem.entity.EntityManager;
+import org.terasology.entitySystem.event.internal.EventSystem;
+import org.terasology.entitySystem.systems.RenderSystem;
+import org.terasology.entitySystem.systems.UpdateSubscriberSystem;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class ComponentSystemManagerTest {
+
+    private ComponentSystemManager sut;
```

Comment:
sut? System Under Testing? I (and most coding academics I imagine) advise against all but the most well known acronyms. I.e. NASA might be appropriate in some contexts, noDISBtET (1) is much less appropriate.

(1) obviously: number of Dogs I Scratched Behind the Ears Today